syftr/flows.py

```diff
@@ -119,6 +120,12 @@ def get_prompt(self, query) -> str:
             few_shot_examples=examples,
         )
 
+    def set_thinking(self, query: str) -> str:
+        if self.use_reasoning is None:
+            return query
+        thinking = "/think" if self.use_reasoning else "/no_think"
+        return f"{thinking} {query}"
```

Comment:
https://huggingface.co/Qwen/Qwen3-8B#advanced-usage-switching-between-thinking-and-non-thinking-modes-via-user-input

AFAIK the typical expectation is that the `/no_think` comes at the _end_ of the prompt